### PR TITLE
Added instructions for installing missing dependencies

### PR DESCRIPTION
**Pull request by jabadge**
This addresses an issue I created for icepack/icepack. I wasn't sure how to check the appearance of the changes I made, but I copied styling from other parts of the page, so I think it's fine.

### CODE REVIEW

File: pages/maintenance.md [hunk 1]
```diff
@@ -40,6 +40,11 @@ Without this flag, you'd have to reinstall again every time you made changes.
 The flag `"[doc]"` at the end tells pip to also install the dependencies you'll need to build the icepack documentation.
 This is part of a [general mechanism](https://setuptools.readthedocs.io/en/latest/references/keywords.html) of Python's setuptools for specifying optional dependencies.
 
+As icepack is updated, you may find that you need to check for and install any missing dependencies. To do this, navigate to the icepack directory and run:
```
Comment (danshapero) [inline pages/maintenance.md:43]: Minor nitpick: I put newlines after every sentence, which doesn't show up in the HTML output but makes version control a lot easier.

**Review by danshapero — CHANGES_REQUESTED** [review]
I have one silly nitpick and that's all!

File: pages/maintenance.md (hunk 1 above)
Comment (jabadge) [reply pages/maintenance.md:43]: Oh right, I knew that, but totally forgot.

Comment (jabadge) [reply pages/maintenance.md]: I fixed this with the update shown below.

**Pull request merged**